backend/app/core/runner.py

```diff
@@ -244,19 +247,19 @@ def run_dataset_validation(dataset_id: str):
         runner_expectation["meta"] = {"expectation_id": expectation.key}
         runner_expectations.append(runner_expectation)
 
-    results = Runner(
+    validation: Validation = Runner(
```

Comment:
The method is correctly typed, so you don't need to manually type-hint the result:

```
validation = Runner(
    ...
).validate()
```